main.js

```diff
@@ -0,0 +1,79 @@
+var lodgeList = [
+  { name: 'Newport Mesa Lodge No. 604',
+    address: '1401 E 15th St  ',
+    city: 'Newport Beach ',
+    state: 'CA, ',
+    zip: '92663',
+    phone: '(949) 515-8788 ',
+    website: 'www.calodges.org/no604',
+    body: 'Blue Lodge',
+    day: 'Wednesday'
+  },
+
+  { name: 'Huntington Beach Lodge No. 380',
+    address: '601 Palm Ave ',
+    city: 'Huntington Beach ',
+    state: 'CA, ',
+    zip: '92648',
+    phone: '(714) 536-3119 ',
+    website: 'www.hbmasons.com',
+    body: 'Blue Lodge',
+    day: 'Wednesday'
+  },
+
+  { name: 'Irvine Valley Lodge #671',
+    address: '23685 Birtcher Dr ',
+    city: 'Lake Forest ',
+    state: 'CA, ',
+    zip: '92630',
+    phone: '(949) 357-0671 ',
+    website: 'www.calodges.org/no604',
+    body: 'Blue Lodge',
+    day: 'Tuesday'
+  },
+
+  { name: 'Orange County York Rite',
+    address: '303 W. Lincoln Ave. ',
+    city: 'Anaheim ',
+    state: 'CA, ',
+    zip: '92805',
+    phone: '(949) 515-8788 ',
+    website: 'www.ocyorkrite.com',
+    body: 'York Rite',
+    day: 'Monday, Saturday'
+  },
+
+  { name: 'Orange County Scottish Rite',
+    address: '303 W. Lincoln Ave. ',
+    city: 'Anaheim ',
+    state: 'CA, ',
+    zip: '92805',
+    phone: '(714) 543-7277 ',
+    website: 'www.ocscottishrite.org',
+    body: 'Scottish Rite',
+    day: 'Monday'
+  }
+]
+
```

Comment:
Good work, let's talk about your rendering strategy though. There's a lot of repetition here. We can talk about leveraging a loop to do this work.